src/components/Home.js

```diff
@@ -0,0 +1,14 @@
+import React, { Fragment } from 'react';
+import '../App.css';
+
+
+const Home = () => {
```

Comment:
can be const Home = () => (<h1 className='home'>Home</h1>)

Reply:
done